infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -63,6 +65,11 @@
      */
     private Map<String, Object> opts;
 
+    /** Handler for this server */
+    @Getter
+    private CorfuMsgHandler handler = new CorfuMsgHandler()
+            .generateHandlers(MethodHandles.lookup(), this);
+
```

Comment:
I see, it might be Amy @amytai , but needs to be clarified. This is as good an opportunity as any, since the PR blames you @no2chem  for this code :)